board/safety/safety_chrysler.h

```diff
@@ -125,12 +125,29 @@ static int chrysler_tx_hook(CAN_FIFOMailBox_TypeDef *to_send) {
   return true;
 }
 
+static void chrysler_init(int16_t param) {
+  chrysler_camera_detected = 0;
```

Comment:
this needs to be set to one camera msgs are on bus 0, right?

Reply:
Yeah, it already is up on line 50:
```
  // check if stock camera ECU is still online
  if (bus == 0 && addr == 0x292) {	  if (bus == 0 && addr == 0x292) {
    chrysler_camera_detected = 1;	    chrysler_camera_detected = 1;
```

Reply:
ah, right!